README.rst

```diff
@@ -34,6 +34,16 @@ command::
 
   $ celery-prometheus-exporter
   Starting HTTPD on 0.0.0.0:8888
+  
+  
+Building
+==========
+Build the container with
+`docker build -f Dockerfile-celery3 -t spothero/redis-celery-exporter:$VERSION --build-arg BUILD_VERSION=$VERSION .`
+replacing `$VERSION` with the version you're building
```

Comment:
What is the current version? Can you provide an example of what VERSION might be?

Reply:
I clarified a bit and formatted. This is a public fork so I don't know to go into too much detail of our internals.